linode/vpcsubnet/resource_test.go

```diff
@@ -100,6 +100,43 @@ func TestAccResourceVPCSubnet_update(t *testing.T) {
 	})
 }
 
+func TestAccResourceVPCSubnet_dualStack(t *testing.T) {
+	t.Parallel()
+
+	resName := "linode_vpc_subnet.foobar"
+	subnetLabel := acctest.RandomWithPrefix("tf-test")
+
+	// TODO (VPC Dual Stack): Remove region hardcoding
+	targetRegion := "no-osl-1"
```

Comment:
There currently isn't a region capability for VPC Dual Stack so it temporarily needs to be hardcoded. This should hopefully be changing soon 👍